content/en/blog/2022/instrument-kafka-clients/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Instrumenting Apache Kafka clients with OpenTelemetry
+linkTitle: Instrumenting Apache Kafka clients with OpenTelemetry
+date: 2022-09-05
+author: "[Paolo Patierno](https://github.com/ppatierno)"
+---
+
+Nowadays, [Apache Kafka](https://kafka.apache.org/) is chosen as the central piece in a distributed system.
+Different services communicate each other by using Apache Kafka as a messaging system but even more as en event or data streaming platform.
+
+Taking into account the cloud-native approach for developing microservices, quite often [Kubernetes](https://kubernetes.io/) is also used to run the workloads.
+In this scenario, you can also easily deploy and manage an Apache Kafka cluster on top of it, by using a project like [Strimzi](https://strimzi.io/).
```

Comment:
strimzi is another cncf project, right?

Reply:
Yes it is. What's the point you would like to make? Maybe that there is an "integration" between Strimzi and OpenTelemetry as part of the CNCF? Adding something like this to the blog post?

Reply:
first of all I wanted to verify that it's another OSS solution and not something commercial.

we might consider cross-posting this blog post in a second step on the cncf blog?

Reply:
We can for sure cross-posting from our Strimzi official web site. We have never published a post on the CNCF blog.